coursedata/adventures/de.yaml

```
@@ -603,164 +609,195 @@ adventures:
+                    Nutze `at random` um einen zufälligen Wert einer Liste zu wählen.
+                    Wenn Du die Zeilen 2 und 3 kopierst und wieder einfügst, kannst Du einen längeren Pfad erzeugen.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
 
                     ```
-                    angles is 10, 50, 90, 150, 250
-                    turn angles at random
+                    Winkel is 10, 50, 90, 150, 250
+                    turn Winkel at random
                     forward 25
                     ```
 
                 start_code: |
-                    angles is 10, 50, 90, 150, 250
-                    turn angles at random
+                    Winkel is 10, 50, 90, 150, 250
+                    turn Winkel at random
                     forward 25
             4:
                 story_text: |
-                    ## Let's draw
-                    In level 4 you have to use quotation marks with `print` and `ask`. Also when drawing!
+                    ## Lass uns zeichnen
+                    In Level 4 musst Du nun auch die Anführungszeichen bei `print` und `ask` nutzen.
+                    Das gilt auch für's Zeichnen!
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    print 'Drawing figures'
-                    angle is 90
-                    turn angle
+                    print 'Zeichne Figuren:'
+                    Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
 
                 start_code: |
-                    print 'Drawing figures'
-                    angle is 90
-                    turn angle
+                    print 'Zeichne Figuren'
+                    Winkel is 90
+                    turn Winkel
                     forward 25
 
             5:
                 story_text: |
-                    ## Let's draw
-                    In level 5 you can make a choice with `if`. For example between different types of figures.
+                    ## Lass uns zeichnen
+                    In Level 5 kannst Du mit dem Befehl `if` Entscheidungen treffen.
+                    Diú kannst Dich zum Beispiel zwischen verschiedenen Formen entscheiden.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    print 'Drawing Figures'
-                    figure is ask 'Do you want a square or a triangle?'
-                    if figure is triangle angle is 120
-                    else angle is 90
-                    turn angle
+                    print 'Zeichne Formen'
+                    Form is ask 'Möchtest Du ein Quadrat oder ein Dreieck zeichnen?'
```

```suggestion
                    Form is ask 'Möchtest Du ein Quadrat oder ein Dreieck?'
```
The user doesn't have to draw. We made them a program that draws for them! :tada: